### DISCUSSION TITLE
為什麼只要勾選「我不是機器人」，Google 就知道你不是機器人？

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#734] [科技新知]
這篇講述了驗證碼的起源以及Google 如何藉此讓所有用戶來訓練AI，話說紅綠燈的桿桿算不算紅綠燈的一部分？我是不是機器人？😢[image](https://user-images.githubusercontent.com/113125794/193766370-7ac5d919-83b0-474c-8f85-b2dd803645df.jpeg)

[文章連結](https://buzzorange.com/techorange/2017/09/13/captcha-and-recaptcha/?fbclid=IwAR3ZXA2Ri1WZ9KRwKur94qF-alTJUqTZQOFNOSwLEgTScaxpJUDANjTt0X0)

**Comment by williamyen0623** [comment]
不用讀懂歪七扭八的驗證碼真的很方便，感謝google大神

**Comment by Zoe06060606** [comment]
原來是透過紀錄各種資料來判斷是不是機器人 
之前一直都好奇他到底怎麼知道的🤣

**Comment by danwan1027** [comment]
原來他是這樣判斷是不是機器人 真的是太酷了

**Comment by Yunhans** [comment]
說不定連機器人都不知道自己是機器人了

**Comment by manipulateed** [comment]
雖然不用輸入驗證碼了，但那個選圖真得更考驗觀察力和儲備知識，心態要很好。

**Comment by HuangHsinTzu** [comment]
直接勾選我不是機器人真的是太方便了
但是再過一陣子會不會又不管用了
可能有人會研發出能片得過這種驗證碼的機器人或程式了

**Comment by Tuoresa** [comment]
非常有趣的一篇文章！
終於知道這個檢驗方式的起源跟原理了
雖然不用輸入驗證碼，但有時候選擇圖片也蠻困難的😂

**Comment by moonnight3503** [comment]
有時候其實圖片驗證也是蠻難的，像我會分不出透抽，哈哈

**Comment by dabbie109303501** [comment]
我之前也有看過這篇文章！
感謝分享

**Comment by ghost** [comment]
原來是透過你打勾前的滑鼠軌跡、網頁捲動紀錄等來判斷啊，我還以為只是做做表面功夫，發明的人真的好厲害啊

**Comment by ching020721** [comment]
原來打勾我不是機器人的背後是這樣的方式去判別我們是不是機器人，雖然有時候判別不出也需要點圖片，希望圖片不要太難不然我就像機器人一樣選不出來哈哈

**Comment by evanchou1204** [comment]
所以如果用手機點他就判斷不出來了QQ 
電腦還比人更判斷的出哪一隻是透抽吧...

**Comment by a45320614** [comment]
真是太酷了 有了noCAPTCHA reCAPTCHA後 我就不會一直因為看不出來透抽而被辨識為機器人了!

**Comment by Rodrigo-dev-bit** [comment]
感謝分享，有多了一個新知識了

**Comment by Pietra1226** [comment]
這篇文超酷~原來是透過追蹤滑鼠箭頭來判斷是不是真人，發明者的逆向思考也太猛了

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
看來我一直以來的問題終於被解決了

**Comment by wenchi9023** [comment]
好酷 終於知道為什麼打勾後有時候會被要求選圖片，有時候又不用！不過蠻好奇打勾前怎樣的滑鼠軌跡和網頁捲動紀錄會被判斷有可能是機器人欸～

**Comment by BlahFish1006** [comment]
好酷! 漲知識了
原來是靠打勾之前的滑鼠軌跡和網頁捲動紀錄等人類行為來判斷是不是機器人
但假如有心人了解了這個機制，又發展出了另一套讓機器人模擬出滑鼠之類的東西，再去隨機捲動網頁、安排隨機路線軌跡點擊，
不知道會不會又被破解了......((黃牛退散

**Comment by Kotori0501** [comment]
感謝分享
不用直接輸入一些驗證碼實在太酷了
因為有時輸入驗證碼時會發生驗證錯誤
但如果是圖形辨認，會方便許多

**Comment by Chiway1229** [comment]
終於知道為什麼以前常看到的輸入字詞的驗證逐漸減少了
原來是這些圖片式的驗證碼取代了舊的模式!

**Comment by Evan0979** [comment]
我之前也超好奇,特別跑去查過XD,好文推推!

**Comment by WEI-CHEN-CHUN** [comment]
第一次知道，但當AI也能透過模擬，這個方式是否會失效

**Comment by peggy0329** [comment]
原來google在驗證碼方面也有一場大戰
透果點選「我不是機器人」抓去以前瀏覽紀錄或其他資料辨別真人是個不錯的方法
不過後續需要再點選圖片的時候，我還很常出錯XD
還蠻好奇如果一直選錯的話，會不會被辨別為機器人或是留下紀錄？

**Comment by ruthchen1020** [comment]
酷!但我其實很想要破解它，然後用來搶票。

**Comment by CPCpyt** [comment]
我覺得這個真的還滿有趣的，
過去也有想過這個問題...
不過後續點選圖片其實沒那麼明顯偶爾也會出！

**Comment by StevenWanglolz** [comment]
原來recaptcha是這樣來的喔~ 還記得之前選紅綠燈選半天，那細細一根到底算不算啊!!! 常常會需要審核，都不知道原來是為了防止有人惡意攻擊

**Comment by swerfg03joe** [comment]
但這種作法早就有人破解了，只需要利用簡單的OpenCV,CNN神經網路分析影像，再結合樹梅派控制機器手臂或是用python控制滑鼠就能騙過驗證碼了

**Comment by ru0372572** [comment]
每次都覺得圖片題很難，超花時間，但驗證碼有時候也會輸錯，啊就長得很像咩

**Comment by Guagua1234** [comment]
我也覺得圖片的很難，
也有可能是因為我不太會分辨，
但這一篇的確有讓我了解到這個系統的運作，
也有可能因為我手機都開無痕，(因為不喜歡每次搜尋都跑出之前很多其他紀錄，沒有色色)
所以每次都要重新按圖片略顯麻煩，更喜歡用驗證碼的方式

**Comment by Valentinaa0214** [comment]
很新奇的小知識！！
原來只需要點選「我不是機器人」，就會傳送一組資料到 Google 的伺服器中，包含 Google 偷偷記錄下來的 IP 位址、國家、時間，以及打勾之前的滑鼠軌跡、打勾之前的網頁捲動紀錄

**Comment by Sylvie0601** [comment]
感謝分享!
每次要用超級難看的驗證碼都會看到生氣，用勾我不是機器人真的超讚，不過希望選圖片不要讓我選超級奇怪的就好了(之前讓我選一個完全沒聽過的海洋動物真的超想砸電腦)。

**Comment by yuxinlin93** [comment]
我以前也很疑惑，為甚麼這樣就知道是不是機器人，就像新手完狼人殺一直喊我是平民一樣，自己說的算。有這種判斷方式真的很方便，不然我之前也遇過很困難的機器人檢測，給我很多房間的照片叫我判斷哪一些是會議室，我怎麼知道大家想在哪一間房間開會???!!!

**Comment by Jzd314** [comment]
可以直接勾選真的很方便，不用一個一個找出正確的圖片或輸入數字，有些圖片或數字是真的很難判斷

**Comment by harrylee1971** [comment]
如果可以破解我是機器人的話，搶五月天的票肯定沒有問題了吧!!!

**Comment by zhiron544** [comment]
我們了解Google如何運用AI來改善驗證碼的使用體驗，也讓我們思考人類和機器人之間的差異和相似之處。

**Comment by Alex11392** [comment]
reCAPTCHA 的這個想法好聰明，而且真的很有意義

**Comment by poiu6fi** [comment]
在做爬蟲的時候，只到遇到那些辨識是不是機器人的東西，像是cloudflare，不管怎麼弄都過不了
